web-wallet/src/lib/stores/walletStore.js

```diff
@@ -201,11 +201,13 @@ export default {
 	getTransactionsHistory,
 	init,
 	reset,
+	set,
```

Comment:
This store was meant to be read only.

Reply:
Are you referring to the mocked version?

Reply:
> Are you referring to the mocked version?

No, I was referring to the real one here.